package.json

```diff
@@ -27,9 +41,8 @@
     "webpack": "4.30.0",
     "webpack-cli": "3.3.12"
   },
-  "name": "salesforcedx-vscode",
   "scripts": {
-    "postinstall": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier",
+    "postinstall": "lerna bootstrap --no-ci -- --no-package-lock && node scripts/reformat-with-prettier && npm run check:peer-deps",
```

Comment:
I was a bit concerned this would be an issue during publish, but I verified we do a `prune` vs a production install so I think we're fine.